src/app/components/forms/post/tags/tags.html

```diff
@@ -0,0 +1,15 @@
+<div v-if="shouldShow">
```

Comment:
Same as game description, maybe just put this in the main component, now that tag-suggestion does most of the work.